### PR TITLE
re: Rewrite `monitor_peers_trigger` in a cleaner way

### PR DESCRIPTION
**Pull request by pmnoxx**
The logic in computing `monitor_peers_trigger` is too complicated.
This PR rewrites logic, to an equivalent logic, that's easier to comprehend.

We can have a discussion, whenever this logic makes sense, but that should be another PR.

TODO:
- [x] Move 10ms to constants
- [x] Move 60s to constants

### CODE REVIEW

File: chain/network/src/peer_manager/peer_manager_actor.rs [hunk 1]
```diff
@@ -1168,7 +1160,13 @@ impl PeerManagerActor {
     ///  - bootstrap outbound connections from known peers,
     ///  - unban peers that have been banned for awhile,
     ///  - remove expired peers,
```
Comment (mm-near) [inline chain/network/src/peer_manager/peer_manager_actor.rs:1162]: add a comment explaining the difference between 'interval', 'default_interval' and 'max_interval

File: chain/network/src/peer_manager/peer_manager_actor.rs [hunk 2]
```diff
@@ -1168,7 +1160,13 @@ impl PeerManagerActor {
...
-    fn monitor_peers_trigger(&mut self, ctx: &mut Context<Self>, max_interval: Duration) {
+    fn monitor_peers_trigger(
+        &mut self,
+        ctx: &mut Context<Self>,
+        mut interval: Duration,
+        default_interval: Duration,
+        max_interval: Duration,
```
Comment (mm-near) [inline chain/network/src/peer_manager/peer_manager_actor.rs:1168]: optional:
we have 3 Duration types next to each other.. 
Maybe we should pass 'default_interval' and 'max_interval' as a tuple?

**Review by mm-near — APPROVED** [review]

File: chain/network/src/peer_manager/peer_manager_actor.rs (hunk 1 above)
Comment (pmnoxx) [reply chain/network/src/peer_manager/peer_manager_actor.rs:1150]: Ok, I added an explanation.

File: chain/network/src/peer_manager/peer_manager_actor.rs (hunk 2 above)
Comment (pmnoxx) [reply chain/network/src/peer_manager/peer_manager_actor.rs]: @mm-near I make it into a tuple. It looks better now.

**Pull request merged**